tests/special_tests.sh

```diff
@@ -19,6 +19,7 @@ python ${DEFAULTS} tests/trainer/optimization/test_manual_optimization.py::test_
 python ${DEFAULTS} tests/models/test_sync_batchnorm.py::test_sync_batchnorm_ddp
 python ${DEFAULTS} tests/plugins/test_deepspeed_plugin.py::test_deepspeed_multigpu
 python ${DEFAULTS} tests/plugins/test_rpc_plugin.py::test_rpc_function_calls_ddp
+# todo (tchaton) tests are hanging in CI.
```

Comment:
A GH issue might be better suited for this TODO

Reply:
Removed !